guava/src/com/google/common/base/Functions.java

```diff
@@ -93,8 +93,7 @@ public String toString() {
     INSTANCE;
```

Comment:
You changed `@org.checkerframework.checker.nullness.qual.Nullable` to `@Nullable` below on line 97/96, but not on the preceding line 92.  I suggest that you do this throughout the codebase, for consistency.

Reply:
Okay. I'll make changes as suggested.